aldryn_people/admin.py

```diff
@@ -37,8 +38,12 @@ def __init__(self, *args, **kwargs):
         user_model = getattr(
             settings, 'AUTH_USER_MODEL', 'auth.User').split('.')
         model = get_model(user_model[0], user_model[1])
-        if model.objects.count() > user_threshold:
-            self.raw_id_fields = ('user', )
+        try:
+            if model.objects.count() > user_threshold:
```

Comment:
I reaaaally don't like this, I saw that you can just use the raw id field widget, maybe try that with formfield_for from admin